specs/electra/beacon-chain.md

```diff
@@ -973,13 +964,14 @@ def verify_and_notify_new_payload(self: ExecutionEngine,
     """
     execution_payload = new_payload_request.execution_payload
     parent_beacon_block_root = new_payload_request.parent_beacon_block_root
-    execution_requests_hash = new_payload_request.execution_requests_hash  # [New in Electra]
+    # [New in Electra]
+    execution_requests_list = get_execution_requests_list(new_payload_request.execution_requests)
 
     # [Modified in Electra:EIP7685]
     if not self.is_valid_block_hash(
             execution_payload,
             parent_beacon_block_root,
-            execution_requests_hash):
+            execution_requests_list):
```

Comment:
I would still prefer to pass `execution_requests` here, and if anything handle the serialization step in the engine API, rather than the consensus specs

Reply:
It’s not about serialization, the exact encoding is currently a part of the protocol and would have to be done even if there was other communication channel than engine API. Serialization of this array to JSON is happening in the engine API client